settings.json

```diff
@@ -1,753 +1,973 @@
+                },
+                {
+                    "name": "Background Gradient",
+                    "class": "background-brand-gradient"
+                },
+                {
+                    "name": "Brand Primary",
+                    "oldNames": ["Primary"],
+                    "class": "background-primary"
+                },
+                {
+                    "name": "Brand Success",
+                    "oldNames": ["Success"],
+                    "class": "background-success"
+                },
+                {
+                    "name": "Brand Warning",
+                    "oldNames": ["Warning"],
+                    "class": "background-warning"
+                },
+                {
+                    "name": "Brand Danger",
+                    "oldNames": ["Danger"],
+                    "class": "background-danger"
+                }
+            ]
+        },
+        {
+            "name": "Shade",
+            "type": "Dropdown",
+            "description": "Apply a shade to your background color",
+            "options": [
+                {
+                    "name": "Light",
+                    "class": "background-light"
+                },
+                {
+                    "name": "Dark",
+                    "class": "background-dark"
+                }
+            ]
+        }
+    ],
+    "Button": [
+        {
+            "name": "Size",
+            "type": "Dropdown",
+            "description": "Size of the buttons",
+            "options": [
+                {
+                    "name": "Small",
+                    "class": "btn-sm"
+                },
+                {
+                    "name": "Large",
+                    "class": "btn-lg"
+                }
+            ]
+        },
+        {
+            "name": "Full width",
+            "type": "Toggle",
+            "description": "Extend the button to the full width of the container it is placed in.",
+            "class": "btn-block"
+        },
+        {
+            "name": "Border",
+            "oldNames": ["Bordered"],
+            "type": "Toggle",
+            "description": "Style the button with a transparent background, a colored border, and colored text.",
+            "class": "btn-bordered"
+        },
+        {
+            "name": "Align icon",
+            "type": "Dropdown",
+            "description": "Place the icon to the right or on top of the button.",
+            "options": [
+                {
+                    "name": "Right",
+                    "class": "btn-icon-right"
+                },
+                {
+                    "name": "Top",
+                    "class": "btn-icon-top"
+                }
+            ]
+        }
+    ],
+    "ListView": [
+        {
+            "name": "Style",
+            "type": "Dropdown",
+            "description": "Change the appearance of rows in the list view.",
+            "options": [
+                {
```

Comment:
Introduce new design property for list view to add horizontal borders at the top and bottom of each list item